src/Akka.Persistence.Sql.Hosting/JournalDatabaseOptions.cs

```diff
@@ -0,0 +1,96 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="DatabaseOptions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Akka.Hosting;
+
+namespace Akka.Persistence.Sql.Hosting
+{
+    public sealed class JournalDatabaseOptions
+    {
+        public static JournalDatabaseOptions Default => new(DatabaseMapping.Default)
```

Comment:
I like the convenience of having ready-made "defaults" for each of the major supported databases here.